Makefile

```diff
@@ -142,7 +142,7 @@ verify-modules: modules  ## Verify go modules are up to date
 .PHONY: verify-gen
 verify-gen: generate  ## Verfiy go generated files are up to date
 	@if !(git diff --quiet HEAD); then \
-		git diff; \
+		git diff | cat; \
```

Comment:
The `cat` ist needed, since otherwise pre-commit waits endlessly for the not visible interactive `git diff` command to get finished.